llvm/test/CodeGen/PowerPC/peephole-combineRLWINM-liveness.ll

```diff
@@ -0,0 +1,111 @@
+; RUN: llc -verify-machineinstrs -mtriple=powerpc-ibm-aix < %s
+; RUN: llc -verify-machineinstrs -mtriple=powerpc64-ibm-aix < %s
+
+; Function Attrs: nocallback nofree nosync nounwind willreturn memory(none)
```

Comment:
I noticed we are not checking anything in this LIT test. Is this intentional, or should we be adding CHECKs?

Reply:
Yeah, maybe some CHECK lines are better. I think the patch is testing the case will not fail for machine verifier.

Can we use MIR tests? The ll case is still too complicated...